bessctl/commands.py

```diff
@@ -1713,34 +1713,34 @@ def get_delta(old, new):
 
     def print_header(timestamp):
         cli.fout.write('\n')
-        cli.fout.write('%-20s%14s%10s%10s        %14s%10s%10s\n' %
-                       (time.strftime('%X') + str(timestamp % 1)[1:8],
-                        'INC     Mbps', 'Mpps', 'dropped',
-                        'OUT     Mbps', 'Mpps', 'dropped'))
+        cli.fout.write('{:<20}{:>14}{:>10}{:>10}        {:>14}{:>10}{:>10}\n'.format(
+                       time.strftime('%X') + str(timestamp % 1)[1:8],
+                       'INC     Mbps', 'Mpps', 'dropped', 'OUT     Mbps', 'Mpps', 'dropped'))
 
```

Comment:
Tiny thing: Could you change `dropped` to `Dropped`?

Reply:
Sure thing